src/commands/commands/agile/agile.test.ts

```diff
@@ -0,0 +1,7 @@
+import { agile } from "./agile";
+
+describe("Agile Command", () => {
+  it("contains correct number of sub-commands", () => {
+    expect(agile.commands).toHaveLength(3);
```

Comment:
I agree. I'll admit that these were kind of stub tests to have _something_ for each of the commands. Since this is a rename and this branch is already getting out of hand, are you ok if I make the updates in a future PR?